internal/logging/zerolog.go

```diff
@@ -58,7 +58,7 @@ func stdoutWriter(truncate bool) io.Writer {
 	writer := zerolog.ConsoleWriter{
 		Out:        os.Stdout,
 		NoColor:    config.InEphemeralClowder(),
-		TimeFormat: time.Kitchen,
+		TimeFormat: "15:04:05",
 	}
```

Comment:
This helps with debugging, seconds were not present and I needed to see how long it took.